enterprise/reporting/error_reporting.go

```diff
@@ -199,6 +217,10 @@ func (edr *ErrorDetailReporter) Report(ctx context.Context, metrics []*types.PUR
 
 		workspaceID := edr.configSubscriber.WorkspaceIDFromSource(metric.ConnectionDetails.SourceID)
 		metric := *metric
+		if !edr.IsPIIReportingDisabled(workspaceID) {
+			edr.log.Debugf("PII setting is disabled for workspaceId: %s", workspaceID)
```

Comment:
We can use a structured log in here.
```suggestion
			edr.log.Debugn("PII setting disabled", obskit.WorkspaceID(workspaceID))
```